pyaerocom/units_helpers.py

```diff
@@ -17,6 +17,20 @@
 
 VARS = const.VARS
 
+
+NON_SI_SUPPORTED = {
+    'concno3' : 'ug N m-3'
+}
+
+NON_SI_UCONV_MUL_FACS = pd.DataFrame([
+    ['concno3', 'ug/m3', 'ug N m-3', 3],
+], columns=['var_name', 'from', 'to', 'fac']).set_index(['var_name', 'from'])
```

Comment:
Can we harmonise this somehow with the already existing `UCONV_MUL_FACS`, that is, that `NON_SI_SUPPORTED` also includes those variables?

Reply:
Oh, forget my comment(s) @ejgal, I confused that we have to go in the other direction here, that is, from standard AeroCom to 'ug N m-3'... now I also understand what you were wondering in `ReadGridded`...